app/tests/unit/components/validators/validators.models.spec.js

```diff
@@ -78,6 +78,12 @@ describe('models.docGen.template', () => {
 
 describe('models.template', () => {
 
+  it('should return false for no filename', async () => {
+    const filename = '';
+    const result = await models.template.filename(filename);
+    expect(result).toBeFalsy();
+  });
+
```

Comment:
Do we have the equivalent checks for when filename is `undefined` and `null`?

Reply:
Added them